README.md

```diff
@@ -1,8 +1,44 @@
-# Typst-Canvas
-A simple [Typst](https://typst.app) library for drawing stuff.
+# Canvas
 
-For more information, see the [manual](manual.pdf).
+Canvas is a simple [Typst](https://typst.app) library for drawing stuff.
 
-## Demos
-### Chart - [PDF](demo/3d-diagram.pdf) - [TYP](demo/3d-diagram.typ)
-[![Bar Chart](demo/3d-diagram.pdf.png)](demo/3d-diagram.typ)
+## Usage
+
+For information, see the [manual](manual.pdf).
+
+### Just
+
+This project uses [just](https://github.com/casey/just), a handy command runner.
+
+You can run all commands without having `just` installed, just have a look into the `justfile`.
+To install `just` on your system, use your systems package manager. On Windows, [Chocolatey](https://chocolatey.org/) (`choco install just`) can be used.
+
+## Installing
```

Comment:
If this is the exact readme that we want to submit there should be something about using the package manager. Espcially as this would be the way the majority of users will be importing the package.